### PR TITLE
Added model dumps

### PR DESCRIPTION
**Pull request by cszsol**

### CODE REVIEW

**Comment by WonderPG** [comment]
This PR was rather to transform the outputs of the tools, not of the app's endpoints. Each tool returns a pydantic class of some sort, and instead of returning the actual pydantic class we should return `model.model_dump()`.

**Comment by kanesoban** [comment]
> This PR was rather to transform the outputs of the tools, not of the app's endpoints. Each tool returns a pydantic class of some sort, and instead of returning the actual pydantic class we should return `model.model_dump()`.

I see. I changed the output of the tools. Let me know if it is ok.

File: swarm_copy/tools/bluenaas_memodel_getall.py [hunk 1]
```diff
@@ -64,4 +64,4 @@ async def arun(self) -> PaginatedResponseUnionMEModelResponseSynaptomeModelRespo
         breakpoint()
```
Comment (BoBer78) [inline swarm_copy/tools/bluenaas_memodel_getall.py:64]: This is probably valid for all the other PRs, this should be removed

**Review by BoBer78 — DISMISSED** [review]
LGTM !

File: swarm_copy/tools/bluenaas_memodel_getall.py (hunk 1 above)
Comment (kanesoban) [reply swarm_copy/tools/bluenaas_memodel_getall.py]: done

**Review by kanesoban — APPROVED** [review]

**Pull request merged**